db/migrate/20220520185517_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :name
+
```

Comment:
The comments table should have column name `text` of `text` data type, `author_id` as a foreign key, and `post_id` as a foreign key.